packages/custom-tooltip-plugin/src/slickCustomTooltip.ts

```diff
@@ -484,6 +518,26 @@ export class SlickCustomTooltip {
     if (finalOutputText.toString()) {
       document.body.appendChild(this._tooltipElm);
 
+      if (this.addonOptions?.persistOnHover === true) {
+        this._tooltipElm.addEventListener('mouseenter', () => {
```

Comment:
let's use my custom service to bind such events, because with my service we can unbind all easily and avoid any event leaking

for example
```
import { BindingEventService } from '@slickgrid-universal/binding';

protected _bindEventService = new BindingEventService();

init() {
  this._bindEventService.bind(editorResetButtonElm, 'click', (e) => console.log('clicked', e));
}

dispose() {
  this._bindEventService.unbindAll();
}
```